lib/rubyflare/connect.rb

```diff
@@ -17,6 +17,7 @@ def initialize(email, api_key)
           http.headers['X-Auth-Email'] = @email
           http.headers['X-Auth-Key'] = @api_key
           http.headers['Content-Type'] = 'application/json'
+          http.headers['User-Agent'] = 'Rubyflare/1.01'
```

Comment:
I think this should use the `VERSION` variable from https://github.com/trev/rubyflare/blob/master/lib/rubyflare/version.rb